packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-summary.js

```diff
@@ -2,13 +2,14 @@
  * WordPress dependencies
  */
 import { __experimentalVStack as VStack } from '@wordpress/components';
+import { PostURLPanel } from '@wordpress/editor';
+
 /**
  * Internal dependencies
  */
 import PageStatus from './page-status';
 import PublishDate from './publish-date';
 import EditTemplate from './edit-template';
-import PageSlug from './page-slug';
 
 export default function PageSummary( {
```

Comment:
Slowly (panel by panel), I'm hoping to unify this component entirely between post and site editors.